### ISSUE TITLE
Error format mixin:

### ISSUE THREAD

**Issue by exdeniz** [ThisIsManta/vscode-stylus-supremacy#1]
```
.test
    +below(1200px)
        display block
```
try format and
```
.test
+below(1200px)
        display block
```

Setting
```
"stylusSupremacy.sortProperties": "grouped",
  "stylusSupremacy.insertColons": false,
  "stylusSupremacy.insertSemicolons": false,
  "stylusSupremacy.insertBraces": false,
  "stylusSupremacy.insertNewLineBetweenSelectors": true,
  "stylusSupremacy.insertLeadingZeroBeforeFraction": false,
  "stylusSupremacy.alwaysUseImport": true,
  "stylusSupremacy.alwaysUseZeroWithoutUnit": true,
  "stylusSupremacy.reduceMarginAndPaddingValues": true,
```

**Comment by ThisIsManta** [comment]
Thank you for reporting. I will take a look ASAP.

**Comment by ThisIsManta** [comment]
Hi @exdeniz,

This issue has been fixed at version **0.2.2**. Please simply update the [extension](https://marketplace.visualstudio.com/items?itemName=thisismanta.stylus-supremacy).
Keep the feedback coming.

Cheers!

**Comment by exdeniz** [comment]
Thanks, it's work!
